spec/components/schemas.yaml

```diff
@@ -1237,6 +1237,10 @@ components:
             - GET
             - POST
           type: string
+        hreflang:
+          description: The language of the link target
+          example: de-CH
+          type: string
```

Comment:
`hreflang` is part of [STAC extension "language"](https://github.com/stac-extensions/language?tab=readme-ov-file#fields-for-links-and-assets). Do we need to point that out somewhere in the documentation?

Reply:
Do you mean for the consumer or rather here for the coder?

Reply:
Possibly both. I wondered whether it needs to be made transparent that it's not vanilla STAC but includes such extensions. For the forecast extension, it was at least a bit visible through the "forecast:" in the field label (could be better though...).

Maybe the reference to the "language" extension is not needed in the docs visible to the consumer, not sure how this is handled. Having a reference to the extension in the code, however, might simplify looking up what that is in the future for a coder.

What do you think?

Reply:
@asteiner-swisstopo @schtibe as noted here https://github.com/radiantearth/stac-spec/blob/master/extensions/README.md#using-extensions stac extenstions should be added to the `stac_extensions` list in principle. I'm unsure for the `hreflang` since we just implement one field and not the whole extension if we should add it or not, I'd rather not. For forecast we can think about it if it gets accepted.